src/main/java/org/jabref/logic/exporter/BibtexDatabaseWriter.java

```diff
@@ -102,9 +102,8 @@ protected void writeString(BibtexString bibtexString, boolean isFirstString, int
                                     .format(bibtexString.getContent(),
                             LatexFieldFormatter.BIBTEX_STRING);
                     getWriter().write(formatted);
-                } catch (IllegalArgumentException ex) {
-                    throw new IllegalArgumentException(
-                            "The # character is not allowed in BibTeX strings unless escaped as in '\\#'.\n" + "Before saving, please edit any strings containing the # character.");
```

Comment:
The `InvalidFieldValueException` actually contained all the necessary information. Moreover, the text was also partly wrong (since the exception could also occur due to a problem with brackets)